### ISSUE TITLE
document.body.scrollTop deprecated

### ISSUE THREAD

**Issue by yosoyadri** [trotzig/react-available-times#5]
I would submit a PR but this is probably a 2 minutes job for you to correct.

In line https://github.com/trotzig/react-available-times/blob/master/src/Day.jsx#L49 to support all browsers and correct/future implementations should be something like:

var realY = pageY - top - (window.pageYOffset || document.documentElement.scrollTop || document.body.scrollTop || 0);

Reference: http://lists.w3.org/Archives/Public/www-style/2013Oct/0287.html

**Comment by trotzig** [comment]
Good catch! Although this would be a quick fix, want to make a PR anyway? The reason I ask is because it would give the project some weight having outside contributors.
